### PR TITLE
Added FAQs to Workshop22 page

### PR DESCRIPTION
**Pull request by JeffCarver**
I added a first draft of answers to the FAQ to the page. @cosden please have a look and see what edits we need to make.

### CODE REVIEW

File: pages/workshop22.md [hunk 1]
```diff
@@ -68,6 +68,22 @@ Across all of the training modules, we have three overaching goals:
 1. Provide attendees with hands-on practice and demonstration
 1. Help demystify buzz words
 
+## FAQs
+#### How much is supposed to be done at the end of the workshop?
+_We aim to have each module at least 90% complete by the end of the workshop._
```
Comment (cosden) [inline pages/workshop22.md]:
```suggestion
_We aim to have each module 90% complete by the end of the workshop.
Additional refinement and improvement can happen asynchronously afterward the workshop_
```

File: pages/workshop22.md [hunk 2]
```diff
@@ -68,6 +68,22 @@ Across all of the training modules, we have three overaching goals:
...
+
+#### What’s the format of the material?
+_We will determine this during the workshop. However we anticipate the materials will reside in some sort of Git repository._
+
+#### What happens after?
+_After the workshop, we will finish curating the materials to prepare for the first [INTERSECT Bootcamp]({{ site.url }}{{ site.baseurl }}/bootcamps/)._
+
+#### Can I teach one of these sections elsewhere?
+_Yes. We intend to create reusable materials that can be taught in other venues._
+
```
Comment (cosden) [inline pages/workshop22.md]:
```suggestion
#### Is there a remote participation option?
_Not at the workshop. However, once the workshop ends we encourage community participation to extend and enhance the training material.
```

File: Gemfile [hunk 3]
```diff
@@ -11,3 +11,5 @@ end
 gem 'asciidoctor', '~> 1.5.4'
 gem 'coderay', '1.1.1'
 
+
+gem "webrick", "~> 1.7"
```
Comment (cosden) [inline Gemfile:15]: What was this for? I pulled it down, installed webrick, and things looked ok locally, so I assume this is fine.

**Review by cosden — CHANGES_REQUESTED** [review]
Added a couple suggestions. I'm a bit nervous about 90%, but I couldn't bring myself to change it. I wanted to write something like 90% of a MVP, with improvements coming later. But it sounded funny.

File: pages/workshop22.md (hunk 1 above)
Comment (JeffCarver) [reply pages/workshop22.md]: Please feel free to edit it to read however you are comfortable. I just put a number there so we would have something.

File: pages/workshop22.md (hunk 2 above)
Comment (JeffCarver) [reply pages/workshop22.md]: This change looks good to me.

File: Gemfile (hunk 3 above)
Comment (JeffCarver) [reply Gemfile:15]: Not sure, when I tried to run it locally, I had to update webrick for it to work. I didn't realize that it changed the gem file.

Comment (cosden) [reply Gemfile:15]: Let me merge this and watch. If something goes wrong I'll back it out.

**Review by cosden — APPROVED** [review]

**Pull request merged**